### DISCUSSION TITLE
https://plugin-components.vuejs.press/zh/guide/media/art-player.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#794] [Announcements]
# https://plugin-components.vuejs.press/zh/guide/media/art-player.html

在 Markdown 文件中嵌入 ArtPlayer。 使用本组件前，请先在你的项目中安装 artplayer 包: 提示 ArtPlayer 组件提供快速插入 ArtPlayer 播放器的能力，如果你需要高级用法你应该阅读 ArtPlayer 文档。 案例 属性 src 类型: string 必填: 是 视频源文件地址。 不支持相对路径。当填写路径名...

https://plugin-components.vuejs.press/zh/guide/media/art-player.html

**Comment by brycenhe** [comment]
安装插件后，在md中使用该代码引入视频，页面上啥也不显示呢？

**Comment by xiangwang1111** [comment]
在md文件中添加最简单的ArtPlayer示例：
<ArtPlayer src="https://vp-demo.u2sb.com/video/caminandes_03_llamigos_720p.mp4" />

报错：
ERROR
t is not a constructor
setup/<@webpack-internal:///./node_modules/vuepress-plugin-components/lib/client/components/ArtPlayer.js:12:3823

请问该怎么处理？
